storybook/src/docs/components/icon-gallery.css

```diff
@@ -0,0 +1,23 @@
+.ams-storybook-icon-gallery {
+  column-gap: var(--ams-space-s);
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(10rem, 1fr));
+  row-gap: var(--ams-space-m);
+}
+.ams-storybook-icon-gallery__item {
+  align-items: baseline;
+  display: flex;
+  flex-direction: column;
+  gap: var(--ams-space-xs);
+
+  > div:first-child {
+    display: flex;
+    gap: var(--ams-space-s);
+  }
+
+  .ams-icon {
+    font-family: var(--ams-typography-font-family);
+    font-size: var(--ams-typography-body-text-large-font-size);
+    line-height: var(--ams-typography-body-text-large-line-height);
```

Comment:
Font family and line height don’t need to be here.

Ideally, we’d find a way to not override font size as well.